paymentsheet/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignup.kt

```diff
@@ -252,11 +268,18 @@ private fun LinkCheckbox(
             enabled = enabled
         )
         Column {
-            Text(
-                text = label,
-                style = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Bold),
-                color = MaterialTheme.colors.onSurface.copy(alpha = contentAlpha)
-            )
+            if (useLinkLogoInCheckboxText) {
+                TextWithLinkLogo(
+                    style = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Medium),
```

Comment:
Reduced the weight a bit so that the Link logo stands out a bit more, matching iOS.